homeassistant/config.py

```diff
@@ -36,6 +36,10 @@
 CONFIG_DIR_NAME = '.homeassistant'
 DATA_CUSTOMIZE = 'hass_customize'
 
+FILE_MIGRATION = [
```

Comment:
may change this in the future, more a fan of an array, since logically to me its not really a key value pair, but a list of tuples.